mlflow/utils/requirements_utils.py

```diff
@@ -0,0 +1,78 @@
+import os
```

Comment:
Can we add some module-level documentation to indicate that this module defines utilities for interpreting and creating requirements files (e.g. pip's `requirements.txt`), which is helpful for managing ML software environments?

Reply:
Got it!